packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -556,8 +556,8 @@ const SubmitError = () => {
         <p>
           If you feel you’ve entered your information correctly, please call the{' '}
           <a href="https://www.va.gov/">VA.gov</a> help desk at{' '}
-          <a href="tel:8555747286">855-574-7286</a>
-          (TTY: 711). We’re here Monday-Friday, 8:00 a.m.-8:00 p.m. ET.
+          <Telephone contact={CONTACTS.VA_311} /> (TTY: 711). We’re here{' '}
+          <abbr title="24 hours a day, 7 days a week">24/7</abbr>.
```

Comment:
@peggygannon, is this ok?

**Before**:

> We’re here Monday-Friday, 8:00 a.m.-8:00 p.m. ET

**After:**

> We're here 24/7

Reply:
Hi, @bkjohnson yes, this is correct styling.